### PR TITLE
Issue 4432 - After a failed online import the next imports are very slow

### PR DESCRIPTION
**Pull request by mreynolds389**
Bug Description:  

When an online import fails the entry and DN caches are "reset", but we use the wrong "new maxsize" which was setting the entry cache maxsize to zero which killed the import performance.

Fix Description:  When resetting the caches use the previous cache maxsize.

Relates: https://github.com/389ds/389-ds-base/issues/4432

### CODE REVIEW

File: dirsrvtests/tests/suites/import/import_test.py [hunk 1]
```diff
@@ -357,6 +355,48 @@ def test_entry_with_escaped_characters_fails_to_import_and_index(topo, _import_c
...
+    bad_import_ldif = ldif_dir + '/perf_import_typo.ldif'
+
+    # Build LDIF file
+    dbgen_users(topo.standalone, 30000, import_ldif, DEFAULT_SUFFIX)
+
+    # Online import which fails
+    import_task = ImportTask(topo.standalone)
+    import_task.import_suffix_from_ldif(ldiffile=bad_import_ldif, suffix=DEFAULT_SUFFIX)
+    import_task.wait()
+
```
Comment (Firstyear) [inline dirsrvtests/tests/suites/import/import_test.py:388]: Does the cache size show up in either the error log or the cn=config that we can check and assert?

Comment (mreynolds389) [reply dirsrvtests/tests/suites/import/import_test.py:388]: It would be under the backend cn=monitor entry as "maxentrycachesize", but after the failed import the backend is disabled and you can not get any cn=monitor data on it.

Comment (Firstyear) [reply dirsrvtests/tests/suites/import/import_test.py:388]: Ahhh okay. Maybe in errorlog? I think we do emit some cache values there ... It'd be good to just have a way to assert the cache number really sticks in the process I think for the test.

Comment (mreynolds389) [reply dirsrvtests/tests/suites/import/import_test.py:388]: We do not "log" the entry cache any where.  The monitor has this info, but in this case its not useful since the backend is offline.  We do report the import cache, but in this case it's the backend entry cache that is the problem.  I'm just a little weary about adding more error logging especially since this (entry cache maxsize) is not something that typically needs to be monitored or exposed in the errors log.  I think this was just an oversight with some of the new backend rework that was done in 1.4.3. 

I know you want to simplify the test case, but I don't want to tweak the server code just to make this corner case bug more easily/reliably detectable.

Comment (Firstyear) [reply dirsrvtests/tests/suites/import/import_test.py:388]: Yeah that's fair. I think I just wanted the test case to "really assert" the fix was correct to prevent regressions, as much as I trust your work. So if you're happy with this "as is" then I think we can look past this.

**Review by progier389 — APPROVED** [review]
LGTM
